src/main/notifications.ts

```diff
@@ -79,8 +79,8 @@ function showNotification(_event: IpcMainInvokeEvent, data: DcNotification) {
       notify.close()
     })
     notify.on('close', () => {
-      notifications[chatId] =
-        notifications[chatId]?.filter(n => n !== notify) || []
+      // this is triggered when the message is moved to notification center
+      // so we don't remove the instance from the list
```

Comment:
Electron docs says:

> Emitted when the notification is closed by manual intervention from the user.
> ~ https://www.electronjs.org/docs/latest/api/notification#event-close

So I think it makes sense to mention in the comment that this is a bug.